tools/cli/pkg/command/reconciliation_info_test.go

```diff
@@ -93,6 +95,41 @@ func TestReconciliationOperationInfoCommand_Run(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "reconciliation info: happy path - reconciliation found",
```

Comment:
Do we check the output of this command or only error?

Reply:
nope. but the logic printing the result to the output is not accessible outside that command to unit test.
That would require some heavy refactor